From 14cd1e1c7d4bc3d6ba4ef70c2c82570fa2c43add Mon Sep 17 00:00:00 2001
From: Matthias <matgnt@gmail.com>
Date: Thu, 28 Jan 2010 20:57:05 +0100
Subject: [PATCH 3/3] no tools because of qt assistant error

---
 src/src.pro |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/src/src.pro b/src/src.pro
index 8dec49b..d50913b 100644
--- a/src/src.pro
+++ b/src/src.pro
@@ -10,7 +10,7 @@ wince*:{
 } else {
     SRC_SUBDIRS += src_corelib src_xml src_network src_gui src_sql src_testlib
     !vxworks:contains(QT_CONFIG, qt3support): SRC_SUBDIRS += src_qt3support
-    include(tools/tools.pro)
+#    include(tools/tools.pro)
     contains(QT_CONFIG, dbus):SRC_SUBDIRS += src_dbus
 }
 win32:SRC_SUBDIRS += src_activeqt
-- 
1.6.3.3

