From 53dccef5285e554d94313c8de22801f67872d09f Mon Sep 17 00:00:00 2001
From: Matthias <matgnt@gmail.com>
Date: Fri, 29 Jan 2010 13:05:02 +0100
Subject: [PATCH 4/5] Disable translations, examples, demos

---
 projects.pro |    6 +++---
 1 files changed, 3 insertions(+), 3 deletions(-)

diff --git a/projects.pro b/projects.pro
index d405a5b..69462bc 100644
--- a/projects.pro
+++ b/projects.pro
@@ -37,9 +37,9 @@ for(PROJECT, $$list($$lower($$unique(QT_BUILD_PARTS)))) {
     isEqual(PROJECT, tools) {
        SUBDIRS += tools
     } else:isEqual(PROJECT, examples) {
-       SUBDIRS += examples
+#       SUBDIRS += examples
     } else:isEqual(PROJECT, demos) {
-       SUBDIRS += demos
+#       SUBDIRS += demos
     } else:isEqual(PROJECT, libs) {
        include(src/src.pro)
     } else:isEqual(PROJECT, docs) {
@@ -50,7 +50,7 @@ for(PROJECT, $$list($$lower($$unique(QT_BUILD_PARTS)))) {
        } else {
           !wince*:!symbian:SUBDIRS += tools/linguist/lrelease
        }
-       SUBDIRS += translations                    # qm build step
+#       SUBDIRS += translations                    # qm build step
     } else:isEqual(PROJECT, qmake) {
 #      SUBDIRS += qmake
     } else {
-- 
1.6.3.3

